docs/Tutorial/4-defining-routes.md

```diff
@@ -0,0 +1,234 @@
+# Creating Routes
+
+In this chapter, you'll learn how to define routes in Fastify.
+We'll set up essential endpoints for our *Quote Vault* API, enabling users
+to store, retrieve, update, and delete memorable quotes.
+
+## find-my-way
+
+Under the hood, Fastify uses [find-my-way](https://github.com/delvedor/find-my-way),
+a very fast, framework-independent HTTP router that relies on a highly efficient
+[**Radix Tree**](https://en.wikipedia.org/wiki/Radix_tree)
```

Comment:
Why does the reader care?

We should specify that the documentation of that module may be useful to understand how to setup complex path parameter or so